### DISCUSSION TITLE
Does Podman allow building different arch images in an ARM host?

### DISCUSSION THREAD

**Discussion by mauromalara** [containers/podman#17939] [Q&A]
I want to run a container from an "amd64" image in macOS with Apple Silicon M2 (ARM), is it possible? If yes, how can I make it?

**Comment by Luap99** [comment]
You can use `podman run --arch amd64 ...` to sepcify the arch of the image. If you use a recent podman version it should support amd64 emulation via qemu-user-static out of the box in the podman machine VM. But keep in mind that emulation is much slower, use the native arch whenever possible.
